Libraries/CMSIS/Device/Maxim/MAX32690/Include/hpb_regs.h

```diff
@@ -229,7 +227,7 @@ typedef struct {
 /**
  * @ingroup  hpb_registers
  * @defgroup HPB_MEMTIM HPB_MEMTIM
- * @brief    Memory timing register.
+ * @brief    Hyperbus emory Timing Register.
  * @{
```

Comment:
Missing an 'M'

Reply:
Whoops, nice catch. I resolved the typo.